doc/examples/datasets_train/datasets_train.py

```diff
@@ -490,6 +620,18 @@ def train_func(config):
     if train.world_rank() == 0:
         return module.cpu()
 
+###############################################################################
+# Input parsing
+# -------------
+#
+# * ``--dir-path <path>``: where we should store our generated parquet data for reading/writing
+# * ``--use-s3``: should we save to S3?
```

Comment:
We'll need to fix this, taking a look at the current code this handles data input + inference output paths, but not the original writing of data.

Reply:
Ah ok, would you be able to run again and fix?